R/addAlpha.R

```diff
@@ -0,0 +1,229 @@
+#' Estimate alpha diversity indices.
+#' 
+#' The function estimates alpha diversity indices optionally using of rarefaction,
+#' then stores results at \code{\link{colData}}.
+#' 
+#' @param x a \code{\link{SummarizedExperiment}} object.
+#' 
+#' @param assay.type the name of the assay used for
+#'   calculation of the sample-wise estimates (default: \code{assay.type = "counts"}).
+#'   
+#' @param index a \code{character} vector, specifying the alpha diversity indices
+#'   to be calculated.
+#'   
+#' @param name a name for the column(s) of the colData the results should be
+#'   stored in. By default this will use the original names of the calculated
```

Comment:
can we use @inheritParams for arguments that are shared with other functions?

Reply:
umm.. I tried to check but couldn't spot a benefit here? since now all the estimate*alpha* functions are transferred to the current implementation? (kinda addAlpha now is the parent function?)

Reply:
Yes, addAlpha is the main function.